common/cmake/devices.cmake

```diff
@@ -1233,7 +1261,30 @@ function(ADD_FPGA_TARGET)
     add_output_to_fpga_target(${NAME} IO_PLACE ${OUT_IO_REL})
     append_file_dependency(VPR_DEPS ${OUT_IO_REL})
 
-    set(FIX_PINS_ARG --fix_pins ${OUT_IO})
+    if(NOT ${NO_PLACE_CONSTR})
+      append_file_dependency(CONSTR_DEPS ${OUT_IO_REL})
```

Comment:
Need to initialize `CONSTR_DEPS` to empty string before appending to it (e.g. `set(CONSTR_DEPS "")`)